docs/guide/migration.md

```diff
@@ -7,11 +7,76 @@ outline: deep
 
 ## Migrating to Vitest 2.0
 
+### Default pool is `forks`
+
+Vitest 2.0 changes the default configuration for `pool` to `'forks'` for better stability. You can read the full motivation in [PR](https://github.com/vitest-dev/vitest/pull/5047).
+
+If you've used `poolOptions` without specifying a `pool`, you might need to update the configuration:
+
+```ts
+export default defineConfig({
+  test: {
+    poolOptions: {
+      threads: { // [!code --]
+        singleThread: true, // [!code --]
+      }, // [!code --]
+      forks: { // [!code ++]
+        singleFork: true, // [!code ++]
+      }, // [!code ++]
+    }
+  }
+})
+```
+
 ### Hooks are running in a stack
 
 Before Vitest 2.0, all hooks were running in parallel. In 2.0, all hooks run serially. In addition to this, `afterAll`/`afterEach` are running in a reverse order.
 
-You can revert to the previous behaviour by changing [`sequence.hooks`](/config/#sequence-hooks) to `'parallel'`.
+You can revert to the previous behaviour by changing [`sequence.hooks`](/config/#sequence-hooks) to `'parallel'`:
+
+```ts
+export default defineConfig({
+  test: {
+    sequence: { // [!code ++]
+      hooks: 'parallel', // [!code ++]
+    }, // [!code ++]
+  },
+})
+```
+
+### `suite.concurrent` runs all its tests concurrently
+
+Previously, specifying `concurrent` on a suite would still group concurrent tests by suites and run them together one by one. Now, it follows jest's behaviour and runs all of them at once (still limited by [`maxConcurrency`](/config/#maxConcurrency))
+
+### enable `coverage.ignoreEmptyLines` by default
+
+Changes default value of `coverage.ignoreEmptyLines` to `true`. This change will have major impact on users' code coverage reports. It is likely that projects using coverage thresholds need to adjust those values after this.
```

Comment:
Might be good to mention that it affects only V8-coverage users. Something like: 

```suggestion
### enable V8 coverage's `coverage.ignoreEmptyLines` by default

Changes default value of `coverage.ignoreEmptyLines` to `true`. This change will have major impact on users' code coverage reports. It is likely that projects using coverage thresholds need to adjust those values after this. This change affects only the default `coverage.provider: 'v8'`.
```